src/modelbench/run.py

```diff
@@ -272,10 +281,8 @@ def print_summary(benchmark, benchmark_scores, anonymize):
         counter = 0
         for bs in benchmark_scores:
             counter += 1
-            key = f"sut{counter:02d}"
-            name = f"System Under Test {counter}"
-
-            bs.sut = FakeSut(key, name)
+            uid = f"sut{counter:02d}"
+            bs.sut = AnonSUT(uid)
```

Comment:
What is this for?

Reply:
It's to anonymize results if someone passes the `--anonymize` option. I don't know if it's actually used, but I don't want to get rid of it because it might be important.